src/pages/licenses/form.tsx

```diff
@@ -41,6 +57,22 @@ export const LicenseForm: React.FC<LicenseFormProps> = ({ defaultDataForm, onSub
     })();
   };
 
+  /**
+  //  * Function to load compliance.
+   */
+  const {
+    data: compliance,
+    isLoading: isLoading,
+    isError: isError,
+    error: error,
+  } = useGetCompliances(auth, queryParams);
+
+  const handleFiltering = useCallback((value: string) => {
+    return setQueryParams((prev) => {
+      return { ...prev, title: "%" + value + "%", page: 1 };
```

Comment:
@ren3459 
* % should not be here. It is server side task.
* can we find single place at project for this task?